src/client/method-status.c

```diff
@@ -266,7 +273,21 @@ static int get_status_unit_on(Client *client, char *node_name, char *unit_name,
         return 0;
 }
 
-static int method_status_unit_on(Client *client, char *node_name, char **units, size_t units_count) {
+
+static int on_unit_status_changed(UNUSED sd_bus_message *m, void *userdata, UNUSED sd_bus_error *error) {
+        units_status *user_data = (units_status *) userdata;
+        printf("\033[2J");
+        printf("\033[%d;%dH", 0, 0);
+        print_info_header(*user_data->max_name_len);
+        get_status_unit_on(
```

Comment:
Just as an FYI: 
Its a bit unfortunate, but I guess we can't avoid having a dedicated call to fetch the latest state since the signal does not contain information about the `enabled` or `freeze` state.